cypress/e2e/submit.cy.ts

```diff
@@ -0,0 +1,79 @@
+describe('Submit page', () => {
+    context('Desktop resolution', () => {
+        beforeEach(() => {
+            cy.viewport(1920, 1080)
+            cy.visit('/submit')
+        })
+
+        it('shows the desktop nav', () => {
+            cy.get('[data-testid="desktop-nav"]').should('exist')
+        });
+
+        it('does not show the hamburger component', () => {
+            cy.get('[data-testid="hamburger-component"]').should('not.exist')
+        });
+
+        it('has a heading', () => {
+            cy.get('h1').should('have.attr', 'data-testid', 'submit-heading').contains('Add a Doctor')
+        });
+
+        it('has a subheading', () => {
+            cy.get('[data-testid="submit-subheading"]').contains(
+                "Want to add a doctor to the database? Complete the form below and it will be reviewed by our moderators.")
```

Comment:
Same as before. Can we import the JSON file that holds the text?

Reply:
Oh, I meant to leave a note that importing the JSON works, but the `$t` doesn't. I'll look into a workaround.